### DISCUSSION TITLE
Copilot with VSCodium is... unstable at best

### DISCUSSION THREAD

**Discussion by UniqueName12345** [community/community#11118] [Copilot Conversations] [answered]
Hi! So, I recently moved from VSCode (you know, the one with bad telementary) to VSCodium (which is VSCode if Microsoft did not make it), and I just noticed that Github Copilot asks for my "Github Personal Token" which did not happen on regular VSCode. Can you guys please fix this, or at least give me a step-by-step tutorial on how to get this "Token". Thank you! 😄

**Comment by davecheney** [comment, marked as answer]
I’m sorry but VSCodium Is not supported. Please use VSCode for best results.

**Reply by UniqueName12345** [reply]
...why?

**Reply by KTibow** [reply]
Since VS Code is made by Microsoft, it has Microsoft features. Although this may include things like telemetry, it also provides things like a dedicated GitHub login system.

I have no idea what the so-called Token could be either, but my best guess is that it might be a PAT, or something provided by the dedicated GitHub VS Code auth system.

**Reply by UniqueName12345** [reply]
> Since VS Code is made by Microsoft, it has Microsoft features. Although this may include things like telemetry, it also provides things like a dedicated GitHub login system.
> 
> I have no idea what the so-called Token could be either, but my best guess is that it might be a PAT, or something provided by the dedicated GitHub VS Code auth system.

Oh! Yeah, this is very old.

**Comment by TheAwesome98-Real** [comment]
Settings > Developer Settings > Personal access token > Generate

unfortunately this does not work yet as authentication fails
